enacts/README.md

```diff
@@ -100,6 +100,25 @@ For final releases of the image, use the `release_container_image` script (no pa
 to build and push to dockerhub.
 
 
+# Running enactstozarr on a partner DL
+
+After having installed a Dockerized system, and after having creating appropriate folders in `/data/datalib/data/` according to the partner's configuration, run the command:
+
+    sudo docker run \
+      --rm \
+      -u $(id -u) \
+      -v /data/datalib/data:/data/datalib/data:rw \
+      -v /usr/local/datalib/build/python_maproom/config.yaml:/app/config.yaml \
+      -e CONFIG=/app/config.yaml \
+      iridl/enactsmaproom \
+      python enactstozarr.py
+
+Should you need to run a modified version of the enactstozarr script, add the option:
```

Comment:
Maybe rephrase "If you need to test a modified version...". I don't want this to be misread as saying that it's ok to use some other script that isn't kept in git. But sometimes we need to make changes to the script in git, and it's nice to be able to test them before committing and building a new container.

Reply:
I just removed it